apps/console/src/features/userstores/components/edit/edit-basic-details-user-store.tsx

```diff
@@ -260,36 +260,50 @@ export const EditBasicDetailsUserStore: FunctionComponent<EditBasicDetailsUserSt
             ]
             : requiredData;
 
-        updateUserStore(id, data).then(() => {
-            patchUserStore(id, patchData).then(() => {
-                dispatch(addAlert({
-                    description: t("adminPortal:components.userstores.notifications." +
-                        "updateUserstore.success.description"),
-                    level: AlertLevels.SUCCESS,
-                    message: t("adminPortal:components.userstores.notifications." +
-                        "updateUserstore.success.message")
-                }));
-                // Prevent update until the userstore update delay is fixed.
-                //update();
-            }).catch(error => {
-                dispatch(addAlert({
-                    description: error?.description
-                        || t("adminPortal:components.userstores.notifications." +
-                            "updateUserstore.genericError.description"),
+        updateUserStore(id, data)
+            .then(() => {
+                patchUserStore(id, patchData)
+                    .then(() => {
+                        dispatch(addAlert<AlertInterface>({
+                            description: t("adminPortal:components.userstores.notifications." +
+                                "updateUserstore.success.description"),
+                            level: AlertLevels.SUCCESS,
+                            message: t("adminPortal:components.userstores.notifications." +
+                                "updateUserstore.success.message")
+                        }));
+
+                        // ATM, userstore operations run as an async task in the backend. Hence, The changes aren't 
+                        // applied at once. As a temp solution, a notification informing the delay is shown here.
+                        // TODO: Remove delay notification once backend is fixed.
```

Comment:
Additionally shall we create an issue and refer it here? So we can track it.

Reply:
Referenced https://github.com/wso2/product-is/issues/9767 here.